internal/telemetry/flags.go

```diff
@@ -0,0 +1,57 @@
+package telemetry
+
+import (
+	"fmt"
+	"strings"
+)
+
+// Mode is the Telemetry Mode
+type Mode string
+
+// String returns the string representation
+func (m Mode) String() string {
+	val := string(m)
```

Comment:
i left you a bad pattern to follow here, but for this we can just `return string(m)` for now (and also make this a one-liner like `Type()` is below)